docs/100_producten/100_borgstelling/010_aanvragen-borgstelling/index.md

```diff
@@ -20,124 +20,39 @@ Voor een maatwerk aanvraag borgstelling kan optioneel een verwijzing naar een pr
+Er is nog geen lopende (aanvraag) borgstelling voor dit kenmerk, de aanvraag moet worden afgehandeld.
 
-<!-- einde -->
-
-* [Pseudo code](RegistreerBorgstellingVerkoopUseCase.java)
-* [Acceptatie criteria](registreer-verkoop.feature)
-
-Zie [exact knowledge base](https://support.exactonline.com/community/s/knowledge-base#All-All-DNO-Content-restapibusinessexamplesalesorder) voor meer informatie.
+### Borgstelling aanvraag afgehandeld
 
-### Borgstelling afgegeven
-
-De gebeurtenis wordt toegevoegd aan de gebeurtenissen verzameling van de kredietbank zodat de gebeurtenis door de kredietbank, decentraal, verder verwerkt kan worden. De status van de borgstelling wijzigt in **afgegeven**.
-
-<!-- einde -->
-
-* [Acceptatie criteria](../verstuur-gebeurtenis.feature)
-
-### Archiveer afwijzing
-
-Nadat de aanvraag is afgewezen wordt er een afwijzing aangemaakt en opgeslagen in het archief. De wettelijke bewaartermijn voor dit soort documenten is 7 jaar.
-
-<!-- einde -->
-
-* [Document](afwijzing.message.md)
-* [Acceptatie criteria](../archiveer.feature)
+De oorspronkelijke borgstelling aanvraag is afgehandeld. Eventuele processen die wachten tot de lopende borgstelling aanvraag is afgerond worden hiermee geinformeerd dat ze verder kunnen gaan in het proces.
 
 ### Borgstelling aanvraag afgewezen
 
-De gebeurtenis wordt toegevoegd aan de gebeurtenissen verzameling van de kredietbank zodat de gebeurtenis door de kredietbank, decentraal, verder verwerkt kan worden. De status van de borgstelling wijzigt in **afgewezen**.
-
-<!-- einde -->
-
-* [Acceptatie criteria](../verstuur-gebeurtenis.feature)
+De gebeurtenis wordt toegevoegd aan de gebeurtenissen verzameling van de kredietbank zodat de gebeurtenis door de kredietbank, decentraal, verder verwerkt kan worden. 
 
-### Archiveer beëindiging
-
-Nadat de aanvraag is beëindigd wordt er een beëindiging aangemaakt en opgeslagen in het archief. De wettelijke bewaartermijn voor dit soort documenten is 7 jaar.
-
-<!-- einde -->
+### Borgstelling afgegeven
 
-* [Document](beeindiging.message.md)
-* [Acceptatie criteria](../archiveer.feature)
+De gebeurtenis wordt toegevoegd aan de gebeurtenissen verzameling van de kredietbank zodat de gebeurtenis door de kredietbank, decentraal, verder verwerkt kan worden. 
 
 ### Borgstelling aanvraag beëindigd
 
-De gebeurtenis wordt toegevoegd aan de gebeurtenissen verzameling van de kredietbank zodat de gebeurtenis door de kredietbank, decentraal, verder verwerkt kan worden. De status van de borgstelling wijzigt in **beëindigd**.
+De gebeurtenis wordt toegevoegd aan de gebeurtenissen verzameling van de kredietbank zodat de gebeurtenis door de kredietbank, decentraal, verder verwerkt kan worden. 
 
-<!-- einde -->
+### Bepaal borgstelling aanvraag resultaat
```

Comment:
@nielserkamp-ritense is deze nog nodig? Via Bepaal aanvraag borgstelling  en Borgstelling aanvraag afgehandeld komt de status toch mee? Volgens mij kan deze weg en kan het proces weer samenkomen voor de end events.

Reply:
Van mij mag hij eruit.

Reply:
Done